### ISSUE TITLE
Add TabBar control

### ISSUE THREAD

**Issue by insinfo** [Microsoft/xaml-standard#128]
A tab bar provides easy access to different views in an app. Use a tab bar to organize information in your app by subtask. The most common way to use a tab bar is with a tab bar controller. You can also use a tab bar as a standalone object in your app.

![tab bars](https://cloud.githubusercontent.com/assets/12227024/26271815/9e5a9042-3ce1-11e7-94b9-9f15f6fca24f.png)

![windows tabcontrol flat design](https://cloud.githubusercontent.com/assets/12227024/26271808/7208f768-3ce1-11e7-911a-594930520356.gif)
<img width="300" alt="1435541252ios_vs_android_tabs" src="https://cloud.githubusercontent.com/assets/12227024/26271810/7fb93800-3ce1-11e7-8146-6e9bd4946a1a.png">
![android-scrollable-tab-layout](https://cloud.githubusercontent.com/assets/12227024/26271812/8a861992-3ce1-11e7-8100-fabd5aa3f995.png)
![tab-bar-icons-iphone-ramotion-animation-interface-design](https://cloud.githubusercontent.com/assets/12227024/26271813/8ef2621a-3ce1-11e7-8fc9-8cf4b4f4740d.gif)

**Comment by mdtauk** [comment]
Don't forget the Windows 10 Tab examples.
![uwp-tab-1](https://cloud.githubusercontent.com/assets/7389110/26275926/ed7fbec2-3d63-11e7-8950-0d14ed7b0459.png)
![uwp-tab-2](https://cloud.githubusercontent.com/assets/7389110/26275927/f0bde230-3d63-11e7-9335-1db43fb28ffa.png)

**Comment by insinfo** [comment]
@mdtauk
Yes very well remembered, I updated with an idea of the syntax including support for the SVG animated icon with SMIL, if you want to contribute syntax as well. And vote for this idea so that it is part of the XAML standard, +1 [Native support of SVG animation with SMIL inside XAML ](https://github.com/Microsoft/xaml-standard/issues/112)

**Comment by birbilis** [comment]
Pivot in WP7/8 was much better than WP10 tabs btw. 

Apart from that in the above design of the phone app, MS forgot a People (Contacts) tab. Glaring omission in practice
